### PR TITLE
Make spillPool a member of MemoryManager

### PR DESCRIPTION
**Pull request by duanmeng**
The spillPool is being held as a static variable in `memory::spillMemoryPool`,
it would be nice to make it a member of the `MemoryManager`
and access through `MemoryManager::spillPool()`.

### CODE REVIEW

File: velox/common/memory/Memory.h [hunk 1]
```diff
@@ -217,6 +217,11 @@ class MemoryManager {
     return *defaultRoot_;
   }
 
+  /// Returns the memory manger's internal spill memory pool for spilling.
```
Comment (xiaoxmeng) [inline velox/common/memory/Memory.h:220]:
```
/// Returns the process wide leaf memory pool used for disk spilling.
```

**Review by xiaoxmeng — COMMENTED** [review]
@duanmeng LGTM. Thanks for the update!

File: velox/common/memory/Memory.h (hunk 1 above)
Comment (duanmeng) [reply velox/common/memory/Memory.h]: Fixed

**Review by xiaoxmeng — APPROVED** [review]
@duanmeng thanks!

**Pull request closed without merging**